src/satosa/frontends/saml2.py

```diff
@@ -512,15 +512,19 @@ def _register_endpoints(self, providers):
         """
         url_map = []
 
+        backend_providers = "|".join(providers)
```

Comment:
It seems the original code dropped the leading `|` from the regexp. It is possible to have empty string in the providers list?